packages/web-demo/src/app/post-sample/components/topic-teaser/topic-teaser-demo/topic-teaser-demo.component.html

```diff
@@ -3,16 +3,16 @@
     <div class="topic-teaser-container bg-nightblue">
       <div class="row pt-huge-r">
 
-        <div class="topic-teaser-image-container col-10 col-rg-8 col-lg-4">
-          <img class="topic-teaser-image" src="assets/images/giraffe.jpg" alt="Giraffe" />
+        <div class="topic-teaser-image-container">
+          <img class="topic-teaser-image" src="https://picsum.photos/id/553/800/800" alt="Giraffe" />
         </div>
 
-        <div class="topic-teaser-content col-12 col-lg-8">
+        <div class="topic-teaser-content">
           <h2 class="topic-teaser-title font-curve-large mb-large">
             <span class="bold">Lorem ipsum</span>
             <span class="light">sit amet, consetetur sadipscing elitr</span>
           </h2>
-          <ul class="link-list font-curve-regular mb-large">
+          <ul class="link-list font-curve-small mb-large">
```

Comment:
The link list font-size should be 20px on desktop:

https://www.figma.com/file/xZ0IW0MJO0vnFicmrHiKaY/Components?node-id=174%3A200

Reply:
True, true...
But there is no matching font-curve for the new topic-teaser design on figma.

| Breakpoint | font-size@Figma | font-size@font-curve-regular | font-size@font-curve-small |
| :-: | -: | -: | -: |
| xs | 14px | 17px | 15px |
| sm | 14px | 17px | 15px |
| rg | 14px | 17px | 16px |
| md | 14px | 17px | 16px |
| lg | 20px | 20px | 17px |
| xl | 20px | 20px | 17px |
| xxl | 20px | 20px | 17px |

Reply:
Do it custom